### PR TITLE
python3Packages.httpagentparser: init at 1.9.1

### PR DESCRIPTION
**Pull request by gador**
Init of httpagentparser

###### Motivation for this change
needed for pgadmin4 init
https://github.com/NixOS/nixpkgs/pull/154764

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage) 
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/httpagentparser/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  version = "1.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "73Y9MZk912GCWs7myLNL4yuVzxZ10cc8PNNfnlKDGyY=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
```
Comment (fabaff) [inline pkgs/development/python-modules/httpagentparser/default.nix:18]: No tests are run.

```suggestion
  checkInputs = [
    pytestCheckHook
  ];
```

File: pkgs/development/python-modules/httpagentparser/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  # PyPi version does not include test directory
```
Comment (fabaff) [inline pkgs/development/python-modules/httpagentparser/default.nix:20]: Perhaps upstream starts to tag the source.

https://github.com/shon/httpagentparser/issues/88

File: pkgs/development/python-modules/httpagentparser/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  doCheck = false;
+
+  pythonImportsCheck = [ "httpagentparser" ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/httpagentparser/";
```
Comment (fabaff) [inline pkgs/development/python-modules/httpagentparser/default.nix:26]:
```suggestion
    homepage = "https://github.com/shon/httpagentparser";
```

The URL on PyPI is not longer valid.

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -3723,6 +3723,8 @@ in {
 
   HTSeq = callPackage ../development/python-modules/HTSeq { };
 
+  httpagentparser = callPackage ../development/python-modules/httpagentparser { };
+
```
Comment (fabaff) [inline pkgs/top-level/python-packages.nix:3727]: Please add the entry to a "alphabetically better position" please.

**Comment by fabaff** [comment]
Please use `python3Packages.httpagentparser: init at 1.9.1` for the commit messges.

File: pkgs/development/python-modules/httpagentparser/default.nix (hunk 2 above)
Comment (gador) [reply pkgs/development/python-modules/httpagentparser/default.nix:20]: thanks for opening the issue upstream!

File: pkgs/top-level/python-packages.nix (hunk 4 above)
Comment (gador) [reply pkgs/top-level/python-packages.nix:3727]: ups. wrong line..

File: pkgs/development/python-modules/httpagentparser/default.nix (hunk 1 above)
Comment (gador) [reply pkgs/development/python-modules/httpagentparser/default.nix:18]: removed

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -4250,6 +4250,16 @@
...
+  gador = {
+    email = "florian.brandes@posteo.de";
+    github = "gador";
+    githubId = 1883533;
+    name = "Florian Brandes";
+    keys = [{
+      longkeyid = "rsa4096/0xBBB3E40E53797FD9";
+      fingerprint = "0200 3EF8 8D2B CF2D 8F00  FFDC BBB3 E40E 5379 7FD9";
+    }];
+  };
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4262]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
commit should be ``python3Packages.httpagentparser: init at 1.9.1``

**Comment by gador** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 155217` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python310Packages.httpagentparser</li>
<li>python39Packages.httpagentparser</li>
</ul>
</details>
